src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -126,15 +128,36 @@ UILabel InitPlaceholderLabel()
 
 			AddSubview(placeholderLabel);
 
-			var edgeInsets = TextContainerInset;
-			var lineFragmentPadding = TextContainer.LineFragmentPadding;
-
-			placeholderLabel.TextInsets = new UIEdgeInsets(edgeInsets.Top, edgeInsets.Left + lineFragmentPadding,
-				edgeInsets.Bottom, edgeInsets.Right + lineFragmentPadding);
-
```

Comment:
What was not working with the existing TextInsets that the new LayoutConstraints fixes?

Reply:
Ditto. Why did the old way not work? And if we need to use constraints, can we create them without using the VFL? That seems like an unnecessary performance hit for some fairly simple constraints.